system.properties

```diff
@@ -19,6 +19,7 @@
 # Tracing if resources are closed/released correctly.
 jvm.resource.tracing=true
 disable.resource.warning=true
+trace.eventLoops= true
```

Comment:
should this be true by default? not sure how/if system.properties are communicated between projects, but it's only necessary to trace event loops if the monitoring / statistics collection for eventloops is turned on in Chronicle-Services

Reply:
either way it'd be good to include the property in systemProperties.adoc as well

Reply:
system.properties is a file in the project and is not released. If you want to set these properties, they need to be set in each repository or project.